makefile

```diff
@@ -50,4 +50,7 @@ doc:
 
 print-%: ; @echo $*=$($*)
 
+test: $(SDC) $(LIBSDRT)
+	cd ./tests; rdmd ./runner.d
```

Comment:
If we are going to use rdmd, could you put it as a shebang in the runner directly and make it executable ?